runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11809,53 +11807,121 @@ static bool inlineIsAssignableFrom(TR::Node *node, TR::CodeGenerator *cg)
    return true;
    }
 
+static TR::SymbolReference *getClassSymRefAndDepth(TR::Node *classNode, TR::Compilation *comp, int32_t &classDepth)
+   {
+   classDepth = -1;
+   // recognizedCallTransformer adds another layer of aloadi
+   while (classNode->getOpCodeValue() == TR::aloadi && classNode->getNumChildren() > 0 && classNode->getFirstChild()->getOpCodeValue() == TR::aloadi)
```

Comment:
Do we need condition `classNode->getNumChildren() > 0` when the node is `aloadi` ?